README.md

```diff
@@ -35,6 +37,8 @@ To get a local copy up and running follow these simple example steps.
 
 ### Usage
 
+Open the index.html in the browser
```

Comment:
- [ ] **OPTIONAL**
It'd be nice if you write more descriptive instructions like:
- open the terminal
- `git clone <repo's link here>`
- open index.html file in the browser
Also, please remove any headlines that are not being used in this project like the Run tests, etc